modules/dkan/dkan_harvest/dkan_harvest.features.field_instance.inc

```diff
@@ -10,6 +10,55 @@
 function dkan_harvest_field_default_field_instances() {
   $field_instances = array();
 
+  // Exported field_instance: 'node-harvest_source-body'.
```

Comment:
@topicus can we reduce the size of this? Also let's put the field at the bottom of the form, so people don't miss the filters.